contents/dataframes_groupby_combine.md

```diff
@@ -1,5 +1,10 @@
 ## Groupby and Combine {#sec:groupby_combine}
 
+### Single Source Columns
+<!-- That fits to the later "Multiple Source Columns" and also solves the
+     "problem" stated in issue #221.
+     (Seems like I have missed this one too ...) -->
+
```

Comment:
```suggestion
```
It's unclear what "problem" you are referring too. I prefer to have text at the start of each section and not start a section immediately with a subsection.